MiniSim/Extensions/UserDefaults+Configuration.swift

```diff
@@ -46,4 +47,9 @@ extension UserDefaults {
         get { bool(forKey: Keys.enableAndroidEmulators) }
         set { set(newValue, forKey: Keys.enableAndroidEmulators) }
     }
+// Need to add ui for changing the terminal preference
```

Comment:
Can you add `TODO: #112 Need to add ui for changing the terminal preference`? I've created an issue for this